src/Defra.PTS.Web.Application/Validation/PetKeeperAddressManualValidator.cs

```diff
@@ -3,48 +3,60 @@
 using Defra.PTS.Web.Domain.ViewModels.TravelDocument;
 using FluentValidation;
 
-namespace Defra.PTS.Web.Application.Validation;
-public class PetKeeperAddressManualValidator : AbstractValidator<PetKeeperAddressManualViewModel>
+namespace Defra.PTS.Web.Application.Validation
 {
-    public PetKeeperAddressManualValidator()
+    public class PetKeeperAddressManualValidator : AbstractValidator<PetKeeperAddressManualViewModel>
     {
-        RuleFor(x => x.AddressLineOne).NotEmpty().WithMessage(x => $"Enter address line 1, typically the building and street");
-
-        When(x => !string.IsNullOrWhiteSpace(x.AddressLineOne), () =>
+        public PetKeeperAddressManualValidator()
         {
-            RuleFor(x => x.AddressLineOne).MaximumLength(AppConstants.MaxLength.AddressLine).WithMessage($"Address Line One must be {AppConstants.MaxLength.AddressLine} characters or less");
-            RuleFor(x => x.AddressLineOne).Matches(AppConstants.RegularExpressions.AddressText).WithMessage("Address Line One is not valid");
-        });
 
-        RuleFor(x => x.TownOrCity).NotEmpty().WithMessage(x => $"Enter Town or City");
-        When(x => !string.IsNullOrWhiteSpace(x.TownOrCity), () =>
-        {
-            RuleFor(x => x.TownOrCity).MaximumLength(AppConstants.MaxLength.TownOrCity).WithMessage($"Town Or City must be {AppConstants.MaxLength.TownOrCity} characters or less");
-            RuleFor(x => x.TownOrCity).Matches(AppConstants.RegularExpressions.AddressText).WithMessage("Town or City is not valid");
-        });
+            RuleFor(x => x.AddressLineOne).NotEmpty().WithMessage("Enter line 1 of your address");
 
-        RuleFor(x => x.Postcode).NotEmpty().WithMessage(x => $"Enter postcode");
-        When(x => !string.IsNullOrWhiteSpace(x.Postcode), () =>
-        {
-            RuleFor(x => x.Postcode).Matches(AppConstants.RegularExpressions.UKPostcode).WithMessage("Postcode is not valid");
-            RuleFor(x => x.Postcode).MaximumLength(AppConstants.MaxLength.Postcode).WithMessage($"Postcode must be {AppConstants.MaxLength.Postcode} characters or less");
-            When(x => x.PostcodeStartsWithNonGBPrefix(), () =>
+            When(x => !string.IsNullOrWhiteSpace(x.AddressLineOne), () =>
             {
-                var validPostcode = false;
-                RuleFor(x => x.Postcode).Must(x => validPostcode).WithMessage("Enter a postcode in England, Scotland or Wales");
+                RuleFor(x => x.AddressLineOne)
+                    .MaximumLength(AppConstants.MaxLength.AddressLine).WithMessage($"Enter line 1 of your address using {AppConstants.MaxLength.AddressLine} characters or less")
+                    .Matches(AppConstants.RegularExpressions.AddressText).WithMessage($"Enter line 1 of your address using {AppConstants.MaxLength.AddressLine} characters or less");
```

Comment:
Regex validation was being removed, what is this regex using?

Reply:
I am validating this on line number 43 that so I removed here